src/order/disjointed.lean

```diff
@@ -7,84 +7,90 @@ import order.partial_sups
 
 /-!
 # Consecutive differences of sets
-
```

Comment:
I think it's usually better to leave these blank lines

Reply:
Whoops, copy-pasting from Github is to blame.